aws/7.4/transitgwyconnect/variables.tf

```diff
@@ -120,6 +120,13 @@ variable "license_type" {
   default = "payg"
 }
 
+// BYOL License format to create FortiGate-VM
```

Comment:
I'd just mention in the docs somewhere that Token is default now.

Reply:
will add in the readme file.